agents-common/src/main/java/org/apache/ranger/plugin/service/RangerBasePlugin.java

```diff
@@ -308,7 +309,7 @@ public void setPolicies(ServicePolicies policies) {
 		if (LOG.isDebugEnabled()) {
 			LOG.debug("==> setPolicies(" + policies + ")");
 		}
-
+		this.serviceConfigs = policies != null ? (policies.getServiceConfig() != null ? policies.getServiceConfig() : new HashMap<>()) : new HashMap<>();
```

Comment:
`this.serviceConfigs = (policies != null && policies.getServiceConfig() != null) ? policies.getServiceConfig() : new HashMap<>();`